### PR TITLE
Poker Format Improvements

### PR DESCRIPTION
**Pull request by bford002**
connects #359 

- converted scores to vertical format and conditionally render a reason
- Added number of estimators that have estimated
- Sorting score in ascending order

Screen Shot 2023-12-05 at 08 52 59 AM

### CODE REVIEW

File: lib/slax_web/websockets/poker.ex [hunk 1]
```diff
@@ -85,13 +85,19 @@ defmodule SlaxWeb.Poker do
...
+        estimates = Enum.map_join(sorted_estimates, ", ", &"#{&1.user}=#{&1.value}")
 
         message = "Estimates for round: #{estimates}"
 
         message =
           if Enum.count(round.estimates, &(!is_nil(&1.reason))) > 0 do
-            reasons = Enum.map(round.estimates, &"#{&1.user} (#{&1.value}): #{&1.reason}")
+            reasons =
+              sorted_estimates
+              |> Enum.filter(&(!is_nil(&1.reason)))
```
[the remark this answers is not in the document]
Comment (bford002) [reply lib/slax_web/websockets/poker.ex:98]: This is only for setting the `reasons` variable. `message` with all the estimates is still put in a string below on line 101. If I'm understanding your concern correctly

[the remark this answers is not in the document]
Comment (bford002) [reply lib/slax_web/websockets/poker.ex:98]: If this helps, we want to reject all estimates below `---` but keep them above
Screen Shot 2023-12-04 at 16 23 18 PM

[the remark this answers is not in the document]
Comment (Dacello) [reply lib/slax_web/websockets/poker.ex:98]: Gotcha, thanks for the visual -- that helps! And looking at it makes me think:

What we just show it like this -- so its all on new lines and we just dont show the desc if there is none:
```
Estimates for the round:
---
user1 (5)
user2 (13): this too hard
```

[the remark this answers is not in the document]
Comment (BunnyDunker) [reply lib/slax_web/websockets/poker.ex:98]: I like this change

[the remark this answers is not in the document]
Comment (bford002) [reply lib/slax_web/websockets/poker.ex:98]: I like it. I'll make that change

**Pull request merged**